src/types/api/competition.ts

```diff
@@ -96,8 +96,9 @@ export interface Competition {
 }
 
 export interface EventRegistration {
+  id: number
   school: SchoolShort
-  grade: string
+  grade: Grade
```

Comment:
mam pocit, ze BE ma niekde dualitu, ze pri GET posiela objekt, alebo pri POST/PATCH caka string ako ID... neviem ci pri tomto, len mi to napadlo 😄